internal/app/decision_maker.go

```diff
@@ -58,20 +58,124 @@ func buildState(s *state) *currentState {
 func (cs *currentState) makePlan(s *state) *plan {
 	p := createPlan()
 
+	var apps map[string]*release
+	if len(s.TargetMap) > 0 {
+		apps = s.TargetApps
+	} else {
+		apps = s.Apps
```

Comment:
This results in not displaying apps not taken into account by -target flag, but we want them to still be displayed so the user knows what's whole scope of the DSF. Why did you add that?

Reply:
my reasoning was that the display on apps we know don't need to be touched is costly -- is there a functional reason to add this in? I'll remove this from the PR shortly either way, just interested

Reply:
the whole idea of DSF is to cover whole environment consisting of multiple apps. Even though you're having -target to one app, this is just a part of the whole env, so we want everyone to know it's still considered among all apps from this env, just marked as ignored, so there's no doubt helmsman sees the env fully

Reply:
ok sure

Reply:
@mkubaczyk fixed